tensorflow/lite/micro/tools/make/ext_libs/cmsis_nn.inc

```diff
@@ -38,12 +46,20 @@ else
 endif
 THIRD_PARTY_CC_HDRS += $(shell find $(CMSIS_NN_PATH)/Include -name "*.h")
 
+ifeq (,$(CMSIS_DSP_LIBS))
+  THIRD_PARTY_KERNEL_CC_SRCS += $(shell find $(CMSIS_DSP_PATH)/Source -name "arm_*.c")
```

Comment:
Not sure if this matters, but this will also grab all the f16 sources as well, which would not be gated on `-DDISABLEFLOAT16`. It might be a safer bet to just list the SRCS from here: https://github.com/ARM-software/CMSIS-DSP/blob/main/Source/Makefile#L35

Reply:
Perhaps it would makes sense to create a new file cmsis_dsp.inc?

Reply:
Ok, I guess it is because we can only specify one optimized kernel dir e.g. OPTIMIZED_KERNELS=cmsis_nn..

Reply:
That's correct.